samples/AutomaticTraceIdInjection/Log4NetExample/Log4NetExample.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace" Version="1.8.0" />
+    <PackageReference Include="Datadog.Trace" Version="1.23.0" />
```

Comment:
Should we update this with our versioning script? (low priority)

Reply:
Done in 6b1c04d2bb480cc906b73c8afe61f3eb1a8a48d8